app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -204,10 +201,7 @@ export default class GalleryPictureModal extends Component<Props, State> {
     const { showMore } = this.state;
 
     return (
-      <Swipeable
-        onSwipingLeft={this.handleSwipeLeft}
-        onSwipingRight={this.handleSwipeRight}
-      >
+      <Swipeable onSwiping={event => this.handleSwipe(event)}>
```

Comment:
`onSwiping={this.handleSwipe}`?

Reply:
Ahh, that seems to do the same 🔥 I'll go with that